src/platform/packages/shared/kbn-tracing-config/src/schema.ts

```diff
@@ -6,9 +6,29 @@
  * your election, the "Elastic License 2.0", the "GNU Affero General Public
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
-import { Type, schema } from '@kbn/config-schema';
+import { type Type, schema } from '@kbn/config-schema';
 import { inferenceTracingExportConfigSchema } from '@kbn/inference-tracing-config';
-import { TracingConfig } from './types';
+import { TracingConfig, type TracingExporterConfig } from './types';
+
+export const tracingExporterConfigSchema: Type<TracingExporterConfig> = schema.oneOf([
+  inferenceTracingExportConfigSchema,
+
+  // GRPC OTLP Exporter
+  schema.object({
```

Comment:
NIT but i am curious why `inferenceTracingExportConfigSchema` is inside the kbn package while these 2 new ones are not?